code/parachain/runtime/composable/src/lib.rs

```diff
@@ -603,6 +603,8 @@ parameter_types! {
 	pub const VaultMinimumDeposit: Balance = 10_000;
 	pub const VaultMinimumWithdrawal: Balance = 10_000;
 	pub const VaultPalletId: PalletId = PalletId(*b"cubic___");
+	pub AssetIdUSDT: CurrencyId = CurrencyId::INVALID;
+	pub FlatFeeUSDTAmount: Balance = 0 * CurrencyId::unit::<Balance>(); //None
```

Comment:
Here we need to whitelist so that we don’t double charge except that the channel isn’t part of the list

Reply:
I think this is for the next task. Here we don't double charge. We charge only on receive always